src/components/admin/WorldNav/WorldNavLabelMap.ts

```diff
@@ -3,7 +3,7 @@ import { WorldNavTab } from "types/world";
 export const WorldNavLabelMap: Readonly<
   Record<WorldNavTab, string>
 > = Object.freeze({
-  [WorldNavTab.general]: "General settings",
+  [WorldNavTab.general]: "Start",
```

Comment:
[NOTE] Well, this is a little LOL, used to be "Start", got renamed to "General settings"...